### ISSUE TITLE
Build error: missing header `instanceNormalizationPluginLegacy.h`

### ISSUE THREAD

**Issue by dbermond** [NVIDIA/TensorRT#4013]
## Description

I'm getting the following build error with TensorRT GA 10.2.0:

```
/build/tensorrt/src/TensorRT/plugin/api/inferPlugin.cpp:35:10: fatal error: instanceNormalizationPlugin/instanceNormalizationPluginLegacy.h: No such file or directory
   35 | #include "instanceNormalizationPlugin/instanceNormalizationPluginLegacy.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

There is no header file named `instanceNormalizationPluginLegacy.h` in the TensorRT repository in version GA 10.2.0. Neither at the `plugin/instanceNormalizationPlugin` directory, or elsewhere.

When trying to remove the `#include` for the missing header, a lot of other errors are shown, like `is not a member`, `no matching function for call` and `invalid template argument`.

## Environment

**TensorRT Version**: GA 10.2.0

**NVIDIA GPU**: RTX 3060

**NVIDIA Driver Version**: proprietary 555.58.02

**CUDA Version**: 12.5.1

**CUDNN Version**: 9.2.0.82

Operating System: Arch Linux

Python Version (if applicable): 3.12.4

Tensorflow Version (if applicable): not applicable

PyTorch Version (if applicable): not applicable

Baremetal or Container (if so, version): not applicable

## Relevant Files

Not applicable.

**Model link**: not applicable

## Steps To Reproduce

```
$ cmake -B build -S TensorRT \
        -G 'Unix Makefiles' \
        -DBUILD_ONNX_PYTHON:BOOL='ON' \
        -DBUILD_SAMPLES:BOOL='OFF' \
        -DCMAKE_BUILD_TYPE:STRING='None' \
        -DCMAKE_INSTALL_PREFIX:PATH='/usr' \
        -DTRT_LIB_DIR="${srcdir}/TensorRT-10.2.0.19/lib" \
        -DGPU_ARCHS='50 52 53 60 61 62 70 72 75 80 86 87 89 90' \
        -DPROTOBUF_VERSION='3.20.1' \
        -Wno-dev
$ cmake --build build
```

**Commands or scripts**: not applicable

**Have you tried [the latest release](https://developer.nvidia.com/tensorrt)?**: yes, version GA10.2.0 does not build, but version GA 10.1.0 compiles fine.

**Can this model run on other frameworks?** For example run ONNX model with ONNXRuntime (`polygraphy run <model.onnx> --onnxrt`): not applicable

**Comment by lix19937** [comment]
I think it maybe a bug.

**Comment by Glat0s** [comment]
I have the same issue when trying to build 10.2.

The file "instanceNormalizationPluginLegacy.h" does not exist in plugin\instanceNormalizationPlugin\

**Comment by lix19937** [comment]
You can try to comment this line, then rebuild.      

> 35 | #include "instanceNormalizationPlugin/instanceNormalizationPluginLegacy.h"  

BTW, I did not verify it.

**Comment by lix19937** [comment]
from  https://github.com/NVIDIA/TensorRT/commit/2332a7191d03e8ec6f85e5d3004803f13659aecf#diff-91f3b74889ce5c76f81833aee176ec5326cbe89f73310c12cad4cea1c8393a60    

![image](https://github.com/user-attachments/assets/e90d404f-53f8-4b11-a2d7-915cd099de33)

**Comment by lix19937** [comment]
@dbermond  @Glat0s  Now it fixed, https://github.com/NVIDIA/TensorRT/pull/4041/files/3cfd5a3e39d4a7ea88b34ba6130d2ae70818d1ec

**Comment by dbermond** [comment]
This is now fixed in version [10.3.0](https://github.com/NVIDIA/TensorRT/releases/tag/v10.3.0). Closing.
